WebApi/WebApiTest/CommentTest.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using WebApi.Models.POCO;
+using Xunit;
+
+namespace WebApiTest
+{
+    public class CommentTest
+    {
+        [Fact]
+        public void Test()
+        {
+            Comment person = new Comment();
+        }
+    }
+}
```

Comment:
Nie rozumiem tej klasy. Jak idzie do pull requesta to do wywalenia, chyba że coś później będzie dodawane to powinna na moje pusta zostać